### PR TITLE
Add database clear method

### PR DESCRIPTION
**Pull request by ovekyc**
database clear method only for test code

### CODE REVIEW

**Comment by fguy** [comment]
https://github.com/Level/levelup#destroy

**Review by fguy — APPROVED** [review]
LGTM

File: src/server/database.js [hunk 1]
```diff
@@ -1,4 +1,10 @@
 import levelup from 'levelup';
 import config from 'config';
+import leveldown from 'leveldown';
 
-export default levelup(config.database, {valueEncoding: 'json'});
```
Comment (fguy) [inline src/server/database.js:4]: I don't think you need to split this line into two lines.

**Comment by fguy** [comment]
Good to have test cases.

File: src/server/database.js (hunk 1 above)
Comment (ovekyc) [reply src/server/database.js:4]: I recognized it and fixed it 
Thank you for FAST review!!!!

**Comment by fguy** [comment]
LGTM

**Comment by hewonjeong** [comment]
LGTM

**Comment by ovekyc** [comment]
The purpose of clearDB function is 
clear all data in database and **reuse**
So, database close and open code was embedded in clearDB function

And add database clear test code

**Comment by festinalente91** [comment]
LGTM

**Pull request merged**